reminders/reminders.go

```diff
@@ -1,27 +1,30 @@
 package reminders
 
 import (
-	"strconv"
+	"context"
+	"fmt"
+	"strings"
 	"time"
 
+	"github.com/botlabs-gg/yagpdb/v2/bot"
 	"github.com/botlabs-gg/yagpdb/v2/common"
 	"github.com/botlabs-gg/yagpdb/v2/common/mqueue"
 	"github.com/botlabs-gg/yagpdb/v2/common/scheduledevents2"
 	"github.com/botlabs-gg/yagpdb/v2/lib/discordgo"
-	"github.com/jinzhu/gorm"
+	"github.com/botlabs-gg/yagpdb/v2/reminders/models"
 	"github.com/sirupsen/logrus"
+	"github.com/volatiletech/sqlboiler/v4/boil"
 )
 
+//go:generate sqlboiler --no-hooks --add-soft-deletes psql
```

Comment:
I was surprised to see that reminders currently soft-delete instead of hard-delete. That is, once triggered, they remain in the database with the `deleted_at` field set. I am unsure whether this is an oversight -- `gorm` does not make it particularly obvious that soft-delete is the default behavior -- or intended.

This PR currently continues to soft-delete, but we could easily switch to hard deletions.